src/dictionaries/en/news.json

```
@@ -21,7 +21,15 @@
   "rendezvenyek": {
     "title": "Events",
     "description": "Events and programs organized by the University Students' Union and student organizations.",
```

Biome (2.5.0)

[error] The key description was already declared.

(lint/suspicious/noDuplicateObjectKeys)